docs/bot-optimization.md

```diff
@@ -67,16 +67,17 @@ update your buy strategy.
 Sample from `user_data/strategies/test_strategy.py`:
 
 ```python
-def populate_buy_trend(self, dataframe: DataFrame) -> DataFrame:
+def populate_buy_trend(self, dataframe: DataFrame, pair: str) -> DataFrame:
```

Comment:
pretty late to the game, but was there a reason not to do e.g.
```
def populate_buy_trend(self, dataframe: DataFrame, **kwargs):
pair = kwargs.get('pair', None)
```

I still have a gut feeling that we will change the function signature in the future anyway..